src/start-sc.ts

```diff
@@ -51,6 +50,16 @@ function buildOptions(): string[] {
 }
 
 export async function startSc(): Promise<string> {
+    try {
+        tmp = mkdtempSync(join(tmpdir(), `sauce-connect-action`))
+        openSync(join(tmp, 'sauce-connect.log'), 'w');
+    } catch (e) {
+        console.error(`Error creating tmp directory for log file: ${e}`);
```

Comment:
You should probably re throw that error after logging

Reply:
updated